test/src/main/resources/scala/test/BUILD

```diff
@@ -1 +1,8 @@
 exports_files(["byes", "hellos", "hellos-and-byes.jar", "more-byes", "more-hellos"])
+
+genrule(
```

Comment:
getting fancy: generated resources...

Reply:
Yeah admittedly this is a subtle corner case... We somehow still managed to hit it though